examples/jdbc-batch-execute-operation/tests/jdbc_batch_execute_operation.bal

```diff
@@ -0,0 +1,32 @@
+import ballerina/test;
```

Comment:
Shall we add license headers for all new files please

Reply:
Added in 011f23d